lib/checkcondition.cpp

```diff
@@ -312,6 +312,13 @@ void CheckCondition::comparison()
                         const std::string& op(tok->str());
                         comparisonError(expr1, expr1->str(), num1, op, num2, false);
                     }
+                    //TRAC #7567: "(a | 7) > 6U" is always true for unsigned a
```

Comment:
I don't like to put ticket number in the checker code. This information is easily available through "git blame".